sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitEditForm.java

```diff
@@ -51,7 +52,7 @@
 	private static final long serialVersionUID = 4265377973842591202L;
 
 	private static final String HTML_LAYOUT =
-		fluidRowLocs(VisitDto.VISIT_STATUS) + fluidRowLocs(VisitDto.VISIT_DATE_TIME, VisitDto.VISIT_REMARKS) + fluidRowLocs(VisitDto.SYMPTOMS);
+		fluidRowLocs(VisitDto.VISIT_STATUS, VisitDto.PERSON) + fluidRowLocs(VisitDto.VISIT_DATE_TIME, VisitDto.VISIT_REMARKS) + fluidRowLocs(VisitDto.SYMPTOMS);
```

Comment:
Please use a custom location (like e.g. GENERAL_COMMENT_LOC in CaseDataForm) instead of VisitDto.PERSON here.